tests/models/bros/test_modeling_bros.py

```diff
@@ -288,6 +289,18 @@ class BrosModelTest(ModelTesterMixin, unittest.TestCase):
         else ()
     )
     all_generative_model_classes = () if is_torch_available() else ()
+    pipeline_model_mapping = (
+        {"feature-extraction": BrosModel, "token-classification": BrosForTokenClassification}
+        if is_torch_available()
+        else {}
+    )
+
+    # BROS requires `bbox` in the inputs which doesn't fit into the above 2 pipelines' input formats.
+    # see https://github.com/huggingface/transformers/pull/26294
+    def is_pipeline_test_to_skip(
+        self, pipeline_test_casse_name, config_class, model_architecture, tokenizer_name, processor_name
+    ):
+        return True
```

Comment:
It's probably better not to add entries into `pipeline_model_mapping` here - by excluding this model in the corresponding pipeline (test) classes.